src/store/index.js

```diff
@@ -635,7 +642,7 @@ export function checkSizeLogic(fieldConfig) {
 
 export function getInheritance(variables) {
   switch (state.componentContext) {
-    case 'Desktop':
+    case 'Desktop': {
```

Comment:
Which rule was forcing this pair of `{ }` to be added?

Reply:
@tonytlwu https://eslint.org/docs/rules/no-case-declarations